docs/development/pigs/pig-006.md

```diff
@@ -0,0 +1,114 @@
+
+# PIG 6 - Data/GTI handling in Gammapy
```

Comment:
@dcfidalgo - Thanks! I'll have a look soon.

Can we have the PIG and PR title be identical?
Maybe "CTA DL3 data handling in Gammapy" or "CTA observation handling in Gammapy"?

I mean, it would be desirable to also have a PIG / plan for slewing telescope observation handling (Fermi-LAT and HAWC, maybe CTA), and to describe all "data handling" along the analysis chain, i.e. how the DL3 EVENTS, IRFs and AUX data goes on as binned maps and spectra and reduced IRFs.

But then this PIG would never finish. I think the scope and content you chose here is good. I'm just suggesting you clarify this a little more in the title and abstract.

Reply:
Ok, i updated the title and made it the same for the PIG and PR.
The abstract now also states more clearly that the focus lies merely on resolving current limitations when analysing IACT data.